.readthedocs.yml

```diff
@@ -0,0 +1,21 @@
+# .readthedocs.yml
```

Comment:
In my experience a `.readthedocs.yml` file is generally less error-prone than `requirements.txt`.

Reply:
I agree